### DISCUSSION TITLE
Adding plans with API rate limit

### DISCUSSION THREAD

**Discussion by justoverclockl** [directus/directus#8993] [Feature Requests]
Would be great if there's a possibility to add a plan to each user groups, for example:

free plan: 100 API request/month
Other plan: 1000 API request/month for X$/month

etc...

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
